airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -717,7 +717,11 @@ def test_defined_cursors_exist_in_schema(self, discovered_catalog: Mapping[str,
         for stream_name, stream in discovered_catalog.items():
             if not stream.default_cursor_field:
                 continue
+
             schema = stream.json_schema
+            if not schema:
```

Comment:
It would be better to move those changes to another PR.

Reply:
I added this change so the PR can run clean CATs, othrewise CI wouldn't pass. Is it OK to keep them?